frontend/src/util/Utility.ts

```diff
@@ -0,0 +1,19 @@
+import SockJS from "sockjs-client";
+import Stomp from "stompjs";
+
+let stompClient:any = null;
+
+export const Connect = (endpoint:string) => {
+  let socket:any = new SockJS(endpoint);
+  stompClient = Stomp.over(socket);
+  stompClient.connect({}, (frame:string) => {
+    console.log('Connected: ' + frame);
+  });
+}
+
+export const Disconnect = () => {
+  if (stompClient !== null) {
+    stompClient.disconnect();
+  }
+  console.log("Disconnected");
+}
```

Comment:
nit: Enable and fix ESLint errors in this file

Reply:
I found the issue; the following had to be added to the `.vscode/settings.json` file:

```
"eslint.workingDirectories": [
  "./frontend"
]
```

I fixed the errors in this file (now `Socket.ts`), and there was also a ["prefer-default-export" error](https://github.com/benmosher/eslint-plugin-import/blob/v2.22.0/docs/rules/prefer-default-export.md) in the `Problem.ts` file that I fixed.